backend/src/routes/api/namespaces/namespaceUtils.ts

```diff
@@ -16,6 +18,36 @@ export const applyNamespaceChange = (
       400,
     );
   }
+  const kc = fastify.kube.config;
+  const cluster = kc.getCurrentCluster();
+
+  const selfSubjectAccessReviewResult = await passThrough<V1SelfSubjectAccessReview>(
+    fastify,
+    request,
+    {
+      url: `${cluster.server}/apis/authorization.k8s.io/v1/selfsubjectaccessreviews`,
+      method: 'POST',
+      requestData: JSON.stringify({
+        apiVersion: 'authorization.k8s.io/v1',
+        kind: 'SelfSubjectAccessReview',
+        spec: {
+          resourceAttributes: {
+            group: 'project.openshift.io',
+            resource: 'projects',
+            subresource: '',
+            verb: 'update',
+            name,
+            namespace: name,
+          },
+        },
+      }),
+    },
+    true,
+  );
+
+  if (!selfSubjectAccessReviewResult.status.allowed) {
+    throw createCustomError('Forbidden', "You don't have the access to update the namespace", 403);
```

Comment:
And same here, should we log here the error? 
Logging failed attempts to update a resource could be useful if we need to debug security issues.

Reply:
Good suggestion, added a log here.